doc/cprover-manual/index.md

```diff
@@ -10,23 +10,28 @@
 [Loop Unwinding](cbmc/unwinding/),
 [Assertion Checking](cbmc/assertions/)
 
-## 4. [Goto-Analyzer &ndash; Abstract Interpretation](goto-analyzer/)
+## 4. [Compositional Reasoning using Code Contracts](contracts/)
 
-## 4. [Test Suite Generation](test-suite/)
+## 5. [JBMC &ndash; Bounded Model Checking Java Bytecode](jbmc-user-manual/)
```

Comment:
As mentioned, I think JBMC is so different that it makes sense to have a separate user manual.